Cargo.toml

```
@@ -103,3 +103,6 @@ isahc = "1"
 mockito = "0.31"
 serde_json = "1"
 tokio = { version = "1", features = ["full"] }
+
+[package.metadata.docs.rs]
+features = ["http-client", "async-http-client"]
```

what do you think about?

```suggestion
all-features = true
```

---

Did not remember that one.